package.org.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
what is this file for?

Reply:
Just the original package.json for comparison reasons. When all things are done it can be deleted.